app/src/main/res/values/strings.xml

```diff
@@ -23,7 +23,7 @@
     <string name="unset">Unset</string>
     <string name="connection_status">Connection status</string>
     <string name="application_icon">application icon</string>
-    <string name="unknown_username">Unknown Username</string>
+    <string name="unknown_username">UNK: %1$s</string>
```

Comment:
because we use [Crowdin](https://crowdin.meshtastic.org/android) for translations, source strings can't be changed or replaced like this. instead, add a new string.

there is also no need to change the translated strings (ie. `values-fr/strings.xml`, etc)